package.json

```diff
@@ -34,6 +34,7 @@
     "displayName"
   ],
   "dependencies": {
+    "babel-helper-annotate-as-pure": "^7.0.0-beta.2",
```

Comment:
You'll want to use `@babel/helper-annotate-as-pure` (we switched to scoped packages)

Reply:
should I update npm package for bable/types also, as now it is not using scoped packages yet

Reply:
To @Andarist's point, we're sneaking in a babel 7 package here and leveraging the fact that it's compatible with Babel 6. I don't think we want to switch to `@babel-types` here yet.

Sidenote, we can probably remove a dep on `babel-types` completely by passing [types down from here](https://github.com/styled-components/babel-plugin-styled-components/blob/master/src/index.js#L7) also.